freqtrade/commands/cli_options.py

```diff
@@ -205,6 +205,11 @@ def __init__(self, *args, **kwargs):
         nargs='+',
         choices=constants.BACKTEST_BREAKDOWNS
     ),
+    "no_backtest_cache": Arg(
+        '--no-cache',
+        help='Do not reuse a cached backtest result even if it exists.',
```

Comment:
```suggestion
        help='Do not reuse a cached backtest result.',
```
the "even if it exists" sounds strange when reading it out of context in a help message (think - backtesting for the first time)